src/SSDT-CD-VSIX/source.extension.vsixmanifest

```diff
@@ -11,9 +11,24 @@
     </Metadata>
     <Installation>
         <!-- "[11.0,16.0)" specifies that this extension is deployable to all versions of VS from 11.0 to 15.999.999, but not 16.0  -->
-        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[15.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Professional" Version="[15.0,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0,17.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[15.0,17.0)">
+            <ProductArchitecture>x86</ProductArchitecture>
```

Comment:
Is there any particular reason to specify the architecture? Can't we just use the previous target IDs and just bump up the version?